### PR TITLE
Cosmetic changes to menus

### PR DESCRIPTION
**Pull request by rita-sonarsource**
image

image

### CODE REVIEW

File: src/Integration.Vsix/Commands/HelpMenu/HelpMenu.vsct [hunk 1]
```diff
@@ -9,10 +11,12 @@
...
     <Buttons>
-      <Button guid="HelpMenuCmdSet" id="ReportProblemCommand" priority="0x100">
-        <Parent guid="HelpMenuCmdSet" id="HelpGroup"/>
+      <Button guid="HelpMenuCmdSet" id="ViewDocumentationCommand" priority="0x100">
+        <Parent guid="HelpMenuCmdSet" id="HelpGroup" />
+        <Icon guid="ImageCatalogGuid" id="StatusHelp" />
+        <CommandFlag>IconIsMoniker</CommandFlag>
         <Strings>
-          <ButtonText>Report a problem</ButtonText>
+          <ButtonText>View Documentation</ButtonText>
```
Comment (rita-sonarsource) [inline src/Integration.Vsix/Commands/HelpMenu/HelpMenu.vsct:19]: Looking at resharper and VS, the help command was always first and with an icon, so I moved it higher
image

File: src/Integration.Vsix/Commands/HelpMenu/HelpMenu.vsct [hunk 2]
```diff
@@ -21,21 +25,19 @@
...
-          <Parent guid="HelpMenuCmdSet" id="HelpGroup" />
-          <Icon guid="guidImages" id="sonarlint" />
-          <Strings>
-            <ButtonText>About SonarLint</ButtonText>
-          </Strings>
-        </Button>
+      <Button guid="HelpMenuCmdSet" id="ReportProblemCommand" priority="0x102">
+        <Parent guid="HelpMenuCmdSet" id="HelpGroup"/>
+        <Strings>
+          <ButtonText>Report a Problem</ButtonText>
```
Comment (rita-sonarsource) [inline src/Integration.Vsix/Commands/HelpMenu/HelpMenu.vsct:31]: Capitalized the P

File: src/Integration.Vsix/Commands/ConnectedModeMenu/ConnectedModeMenu.vsct [hunk 3]
```diff
@@ -13,7 +13,7 @@
         <Parent guid="ConnectedModeMenuCmdSet" id="ConnectedModeGroup"/>
         <Icon guid="guidIcons" id="bmpSonarQube"/>
         <Strings>
-          <ButtonText>Manage Server Connections...</ButtonText>
+          <ButtonText>Bind in SonarQube\SonarCloud</ButtonText>
```
Comment (rita-sonarsource) [inline src/Integration.Vsix/Commands/ConnectedModeMenu/ConnectedModeMenu.vsct:16]: I still don't like it :/
image

Comment (georgii-borovinskikh-sonarsource) [reply src/Integration.Vsix/Commands/ConnectedModeMenu/ConnectedModeMenu.vsct:16]: Bind to?

**Review by georgii-borovinskikh-sonarsource — APPROVED** [review]

File: src/Integration.Vsix/Commands/ConnectedModeMenu/ConnectedModeMenu.vsct (hunk 3 above)
Comment (bigfluffycookie) [reply src/Integration.Vsix/Commands/ConnectedModeMenu/ConnectedModeMenu.vsct:16]: Wouldnt it normally be forward slash?

Comment (rita-sonarsource) [reply src/Integration.Vsix/Commands/ConnectedModeMenu/ConnectedModeMenu.vsct:16]: I wondered about the slash too, I couldn't find any examples

**Pull request merged**